### DISCUSSION TITLE
insight/quickstart/otel/golang-ebpf/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5339] [Announcements]
# insight/quickstart/otel/golang-ebpf/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/insight/quickstart/otel/golang-ebpf/

**Comment by wilsonxstsai** [comment]
有一个问题，基于 eBPF 的无侵入埋点目前应用场景很受限吧，我看了 opentelemetry-go-instrument 这个官方的项目，官方的说明不包含 Redis 库，我自己测试了也无法埋点，不知道你们有没有基于这个项目做定制的框架开发？

**Reply by JaredTan95** [reply]
目前没有投入定制化。
